components/avatar/overview.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Overview
+page_title: Avatar Overview
+description: Discover the Blazor Avatar and explore the examples.
+slug: avatar-overview
+tags: telerik,blazor,avatar,overview
+published: True
+position: 0
+---
+
+# Blazor Avatar Overview
+
+The <a href="https://www.telerik.com/blazor-ui/avatar" target="_blank">Blazor Avatar component</a> is typically used to display images, icons or initials representing people or other entities.
+
+It also gives you the flexibility to customize its size, color, fill mode and more.
+
+## Creating Avatar
+
+1. Use the `TelerikAvatar` tag to add the component to your razor page.
+
+1. Declare your desired content inside the `TelerikAvatar` tag - text, icon or image.
+
+1. Set the [Avatar type]({%slug avatar-types%}) based on your content.
```

Comment:
This step is not actionable unless the reader refers to other docs content. In such cases, we should either add one more sentence for clarification, or just suggest a specific setting for now and leave the other options for later.